Classes/common/CDTIncrementalStore.h

```diff
@@ -0,0 +1,121 @@
+//
+//  CDTIncrementalStore.h
+//
+//
+//  Created by Jimi Xenidis on 11/18/14.
+//
+//
+
+#import <CoreData/CoreData.h>
+#import <CloudantSync.h>
+
+extern NSString *const kCDTISErrorDomain;
+extern NSString *const kCDTISException;
```

Comment:
ACK... probably :+1: 
I actually switched back and forth based on my experience with existing code.
Both cloudant (see `CDTIndexManager.[hm]`) and Cocoa use it a lot.
It is not mentioned in the Apple [Coding Guidelines for Cocoa](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingIvarsAndTypes.html).
Apparently it is Hungarian Notation.
Personally, I don't care as long as it's consistent and follows some prescribed style guide and since they quote the cocoa style.. I shall remove them.